src/components/EditProfile.js

```diff
@@ -0,0 +1,87 @@
+              <h4 className="card-title">Edit Profile  -
+                <small className="category">  Complete your profile</small>
+              </h4>
+              <form>
+                <div className="row">
+                  <div className="col-md-3">
+                    <div className="form-group label-floating is-empty">
+                      <label className="control-label">Username</label>
+                      <input type="text" className="form-control" value={this.state.username} onChange={(e) => this.setState({username: e.target.value})}/>
+                    <span className="material-input"></span></div>
+                  </div>
+                  <div className="col-md-4">
+                    <div className="form-group label-floating is-empty">
+                      <label className="control-label">Email address</label>
+                      <input type="email" className="form-control" value={this.state.email} onChange={(e) => this.setState({email: e.target.value})}/>
+                    <span className="material-input"></span></div>
+                  </div>
+                  </div>
+                  <div className="row">
+                    <div className="col-md-6">
+                      <div className="form-group label-floating is-empty">
+                        <label className="control-label">First Name</label>
+                        <input type="text" className="form-control" value={this.state.firstName} onChange={(e) => this.setState({firstName: e.target.value})}/>
+                      <span className="material-input"></span></div>
+                    </div>
+                    <div className="col-md-6">
+                      <div className="form-group label-floating is-empty">
+                        <label className="control-label">Last Name</label>
+                        <input type="text" className="form-control" value={this.state.lastName} onChange={(e) => this.setState({lastName: e.target.value})}/>
+                      <span className="material-input"></span></div>
+                    </div>
+                  </div>
+                  <div className="row">
+                    <div className="col-md-12">
+                      <div className="form-group">
+                        <label>About Me</label>
+                        <div className="form-group label-floating is-empty">
+                          <textarea className="form-control" rows="5" value={this.state.bio} onChange={(e) => this.setState({bio: e.target.value})} ></textarea>
+                        <span className="material-input"></span></div>
+                      </div>
+                    </div>
+                  </div>
+                  <button type="submit" className="btn btn-rose pull-right">Update Profile</button>
+                  <div className="clearfix"></div>
+              </form>
+            </div>
+          </div>
+        </div>
+    </div>
+    );
+  }
+}
+
+export default EditProfile;
```

Comment:
Remember that you need to wrap components that need to be rendered in the react-router pipeline with `withRouter()` even if you're not mapping to props etc..

Reply:
will do